lib/components/ChannelView.jsx

```diff
@@ -61,21 +61,38 @@ const ChannelView = ({
 }) => {
     const [open, setOpen] = useState(true);
 
+    const channelFromValue = value => {
+        if (value === 0) {
+            return 37;
```

Comment:
Too many [magic numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) IMO, especially the channel numbers. Consider replacing with named constants or adding comments.

Reply:
Agree. Will find a solution for it.